conformance/reports/README.md

```diff
@@ -62,7 +62,7 @@ is structured as follows:
   needs to add information in the "reproduce" section (see below) on how to configure
   such a mode.
 - **Report**: the link to the related report. It MUST be in the form of
-  `[link](./report.yaml)`. The reports must be named according to the following
+  `[_version_ report](./report.yaml)`. The reports must be named according to the following
```

Comment:
This changes the instructions which should help with future pull requests, but, indeed a linter is necessary.

I'm, of course, using one, and if people like the results, I'm happy to talk with them about it, but it's outside the scope of this PR.